identifier/identifier.go

```diff
@@ -22,11 +28,20 @@ type ACMEIdentifier struct {
 	Value string `json:"value"`
 }
 
-// DNSIdentifier is a convenience function for creating an ACMEIdentifier with
-// Type DNS for a given domain name.
-func DNSIdentifier(domain string) ACMEIdentifier {
+// NewDNS is a convenience function for creating an ACMEIdentifier with Type
+// NewDNS for a given domain name.
```

Comment:
```suggestion
// DNS for a given domain name.
```

Reply:
Ooh good catch. I've always been so happy that vscode's auto-rename feature also updates doc comments but I'll have to watch out for this in the future.